defs/support.json

```diff
@@ -1,100 +1,245 @@
+        }
+    },
+    "btc:TAZ": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "webwallet": true
+    },
+    "btc:TBCH": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "webwallet": null
+    },
+    "btc:TDCR": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": null,
+        "webwallet": true
+    },
+    "btc:TEST": {
+        "connect": true,
+        "electrum": null,
+        "trezor1": "1.5.2",
+        "trezor2": "2.0.5",
+        "webwallet": true
+    },
+    "btc:TLTC": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "webwallet": true
+    },
+    "btc:VIA": {
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "connect": null,
+        "webwallet": true,
+        "other": {
+            "Vialectrum": "https://vialectrum.org"
+        }
+    },
+    "btc:VTC": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.1",
+        "trezor2": "2.0.5",
+        "webwallet": true
+    },
+    "btc:XZC": {
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "connect": null,
+        "webwallet": true,
+        "other": {
+            "Electrum-XZC": "https://github.com/zcoinofficial/electrum-xzc",
+            "Znode Tool": "https://github.com/yura-pakhuchiy/znode-tool"
+        }
+    },
+    "btc:ZEC": {
+        "connect": true,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "webwallet": true
+    },
+    "btc:ZEN": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": null,
+        "trezor2": "2.0.8",
+        "webwallet": null
+    },
+    "btc:tDASH": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": null,
+        "webwallet": null
+    },
+    "btc:tGRS": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.8",
+        "webwallet": true
+    },
+    "btc:tXZC": {
+        "connect": null,
+        "electrum": null,
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "webwallet": null
+    },
+    "network:LSK": {
+        "trezor1": null,
+        "trezor2": "2.0.7",
+        "connect": null,
+        "webwallet": null
+    },
+    "network:XLM": {
+        "trezor1": "soon",
+        "trezor2": "soon",
+        "connect": null,
+        "webwallet": null
+    },
+    "network:XTZ": {
+        "trezor1": null,
+        "trezor2": "soon",
+        "connect": null,
+        "webwallet": null
+    },
+    "network:ADA": {
+        "trezor1": null,
+        "trezor2": "soon",
+        "connect": null,
+        "webwallet": null
+    },
+    "network:XMR": {
```

Comment:
Not sure about the `network` prefix, but as I cannot find a better word, let's keep it for now I guess.